examples/lua_REPL/README.md

```diff
@@ -0,0 +1,20 @@
+### About
```

Comment:
Could you provide a readme with the available commands? I ran the example but don't know what to do :)

Reply:
@jcarrano pointed out that it's not our place to teach people to program in Lua (we wouldn't link to a JavaScript tutorial in the jerryscript pkg for example). Worth noting that at the moment we haven't done any bindings. The only officially supported function is print() as far as I'm aware

Reply:
Link to the lua page and some brief info "works in the same way as the lua shell in the default installation"